udata/core/dataset/rdf.py

```diff
@@ -312,6 +312,17 @@ def temporal_from_rdf(period_of_time):
         log.warning('Unable to parse temporal coverage', exc_info=True)
 
 
+def contact_point_from_rdf(rdf):
+    contact_point = rdf.value(DCAT.contactPoint)
+    if contact_point:
+        name = contact_point.value(VCARD.fn)
+        email = (contact_point.value(VCARD.hasEmail)
+                 or contact_point.value(VCARD.email)
+                 or contact_point.value(DCAT.email))
+        contact = ContactPoint(name=name, email=email).save()
```

Comment:
We should either get or create the contact point from the owner/organization here.